DanylkivPavlo/HW6/task1.py

```diff
@@ -0,0 +1,18 @@
+even_num_dis_2 = []
```

Comment:
output should be the next
[2, 4, 6, 8, 10] is even multiple of 2
[3, 9] is an odd multiple of 3
[1, 5, 7] not divisible by 2 and 3

look at this code
l1 = [ x for x in range(1, 11) if x % 2 == 0 ]
l2 = [ x for x in range(1, 11) if x % 2 == 1 and x % 3 == 0 ]
l3 = [ x for x in range(1, 11) if not x % 2 == 0 and not x % 3 == 0 ]
print(l1, 'is even multiple of 2')
print(l2, 'is an odd multiple of 3')
print(l3, 'not divisible by 2 and 3')